book/plugins/general-abilities/multiple-types-one-repo/multiple-types-one-repo.md

```diff
@@ -0,0 +1,15 @@
+# One Code Base for Multiple Plugins
+
+A developer can open one repository for handling different plugin types.
```

Comment:
Use not open.
Also repository is the wrong term here.
Codebase, Library , Framework , target ... choose the most generic one

Reply:
Also I would mention - The main difference between plugin types and abilities is the fact a single plugin can implement multiple abilities but can be of one type.
To read more about the different plugin types click here (add link)
To read more about the different plugin abilities click here (add link)

Reply:
Done